user_guides/jsonb/pom.xml

```diff
@@ -47,7 +47,7 @@
 
     <distributionManagement>
         <site>
-            <url>scm:git:git@github.com:javaee/tck_jsonb_v1_0.git</url>
+            <url>scm:git:git@github.com:eclipse-ee4j/jakartaee-tck.git</url>
         </site>
     </distributionManagement>
 
```

Comment:
Copyright year needs to be updated in this file from Copyright (c) 2017, 2019 to Copyright (c) 2017, 2020.

Reply:
Fixed.